src/components/Contacts/ContactFlow/ContactFlowSetup/Header/ResetToDefault/ResetToDefaultModal.tsx

```diff
@@ -42,13 +43,15 @@ export const ResetToDefaultModal: React.FC<ResetToDefaultModalProps> = ({
 }) => {
   const { t } = useTranslation();
   const { getContactStatusesByPhase } = useContactPartnershipStatuses();
+  const phases = useApiConstants()?.phases;
```

Comment:
Could you write this? this might cause an error though. you would need to test it
```suggestion
  const {phases} = useApiConstants();
```

Reply:
Yeah I tried this and it doesn't work.